spec/services/products_renderer_spec.rb

```diff
@@ -155,6 +155,19 @@
           products = products_renderer.send(:products)
           expect(products).to eq([product_cherries, product_banana_bread, product_doughnuts])
         end
+
+        it "filters products with producer properties when sorting is enabled" do
+          allow(distributor).to receive(:preferred_shopfront_taxon_order) {
+            "#{fruits.id},#{cakes.id}"
+          }
+          fruits_supplier.producer_properties.create!({ property_id: property_organic.id,
+                                                        value: '1', position: 1 })
+          search_param = { q: { "with_variants_supplier_properties" => [property_organic.id] } }
+          products_renderer = ProductsRenderer.new(distributor, order_cycle, customer, search_param)
+
+          products = products_renderer.send(:products)
```

Comment:
According to our rubocop rules, this needs to use the `__send__` method.
Rubocop was skipping this file, but we recently enabled it (after this branch was made).
I will go ahead and correct it.
```suggestion
          products = products_renderer.__send__(:products)
```

Reply:
It's worth noting that usage of `send`/`__send__` is generally not good practice, and we should be testing the result of the public methods. So next time, we should try to do that. This is fine for now though.